Governance/retire/retire-replica.md

```diff
@@ -11,23 +11,23 @@ This is because it is hard in the first place to choose which data, or data subs
+* **Checking latest access time** (`atime`) of the files (using the `stat` command) to see when a file was last accessed. Note that many operations can affect `atime` so it's important to do this before moving the data anywhere. This is only reliable if there are not automated processes that alter `atime` when running, for example a regular scan of the filesystem run by the system administrator. This can only say when a file was last accessed not how many users are accessing it. 
+* **Project user number:** where users need to sign up to use the data (as for NCI project) there is a record of users' number and distribution. A limit of this is that it doesn't say who is actively using the data or which subset of the data is being used.
 * **Database queries:** this can be used of course only if the data can be queried or accessed via a tool and this tool keeps logs of the queries. It is still imprecise as a user might be querying some data but not necessarily using it.
-* **Online downloads:** usually data servers keep tracks of downloads of online data, as well as views. If the data is on Zenodo these stats are accessible to anyone, in other cases you might have to ask the data server administrators to collect them for you, if they don't do that automatically.
+* **Online downloads:** usually data servers keep track of downloads of online data, as well as views. If the data is on Zenodo these stats are accessible to anyone, in other cases the data server administrators might have to collect them for you, if they don't do that automatically.
 * **Automated download of updated datasets:** when a dataset is maintained from a 3rd party host automatically, then new versions can be automatically detected and downloaded and older versions or data with identified serious errata removed. This process is in place for NCI's CMIP replica data using the `synda` tool.
 
 ### Community feedback
-* **E-mail communication:** this is possible where your community is well defined, so if users need to sign up to a group/project or if there are well defined community of potential users. For example, if the data is on NCI, you can use my.nci.org.au to contact projects' users. This might be sufficient when you have already identified a likely candidate for removal.
-* **Survey:** this can be useful when you need to evaluate several datasets at one time or work out which subset of a big dataset could be retired.
+* **E-mail communication:** this is possible where the community is well defined, such as if users need to sign up to a group/project or if there are a well defined community of potential users. For example, if the data is on NCI, my.nci.org.au can be used to contact projects' users. This might be sufficient when a likely candidate for removal has already been identified.
```

Comment:
```suggestion
* **E-mail communication:** this is possible where the community is well defined, such as if users need to sign up to a group/project or if there are a well defined community of potential users. For example, if the data is on NCI, Mancini ([http://my.nci.org.au](my.nci.org.au)) can be used to contact projects' users. This might be sufficient when a likely candidate for removal has already been identified.
```

Reply:
I'm not 100% sure I got the format right for that URL, if you're happy to have the HTTP in text you can just write the whole thing once and drop the brackets.